src/autopas/containers/octree/OctreeDirection.h

```diff
@@ -0,0 +1,532 @@
+/**
+ * @file OctreeInnerNode.h
```

Comment:
this stuff feels so special I think it should go in a dedicated namespace. E.g. `OctreeDirection` Don't forget to also place the namespace's doc in `namespaces.h`